pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Debian supports wildcard versions. That can help with unattended upgrades. Does CentOS support wildcards as well?

If so, this change should supports wildcards.